spec/models/need_spec.rb

```diff
@@ -377,4 +284,213 @@
 
     it { is_expected.to eq 'taking_care' }
   end
+
+  describe 'paniers relance' do
+    describe 'besoins à relancer (J+7)' do
+      # - besoins restés sans réponse (=sans positionnement, personne a cliqué sur les boutons "je prends en charge" ou "je refuse") à plus 7 jours après les mises en relation ;
+      # - besoins avec une mise en relation clôturée par « pas d’aide disponible » et « non joignable » ou refusés ET pour lesquels des experts n’ont toujours pas répondu à plus de 7 jours.
+
+      describe 'contraintes de délais' do
+        # DELAIS
+        # - besoin créé il y a 06 jours, sans positionnement     ko
+        # - besoin créé il y a 07 jours, sans positionnement     ok
+        # - besoin créé il y a 13 jours, sans positionnement     ok
+        # - besoin créé il y a 14 jours, sans positionnement     ko
+
+        let(:reference_date) { Time.zone.now.beginning_of_day }
+
+        let(:need1) { travel_to(reference_date - 6.days)  { create :need_with_matches } }
+        let(:need2) { travel_to(reference_date - 7.days)  { create :need_with_matches } }
+        let(:need3) { travel_to(reference_date - 13.days) { create :need_with_matches } }
+        let(:need4) { travel_to(reference_date - 14.days) { create :need_with_matches } }
+
+        before do
+          need1
+          need2
+          need3
+          need4
+        end
+
+        it 'displays needs in correct time_range' do
```

Comment:
ah et ici c'est en anglais 🙂

Reply:
comment ça, c'est pas cohérent ;) ?
hmm, là en tout cas, je voyais pas comment faire autrement, même si les `it` me viennent + naturellement en anglais. 

J'essaye d'homogénéiser ça. 

Par contre, j'arrive pas encore à juger de la pertinence de tout passer en français. Il y aura peut-être une distinction à faire suivant qu'on est sur des tests métiers ou pas ? (par exemple, des "sous scope" utilisés dans des scope métiers)

On se repose la question lors de la rédaction de prochains tests ?